### ISSUE TITLE
Fix duplicate ID attributes

### ISSUE THREAD

**Issue by atsansone** [flutter/website#9529]
Fix any and all duplicate ID attributes. These appear mostly on the sidenav. This violates WCAG[F77](https://www.w3.org/TR/WCAG20-TECHS/F77).

**Error:** Duplicate id attribute value "sidenav-27" found on the web page.
[htmlcs: WCAG2AA.Principle4.Guideline4_1.4_1_1.F77](https://squizlabs.github.io/HTML_CodeSniffer/Standards/WCAG2/#sc4_1_1)

**Example:**

```html
<ul class="nav flex-column flex-nowrap collapse " id="sidenav-27"> <li class="nav-hea...</ul>
```

**Location:**
(selector "#sidenav-27")

**Comment by parlough** [comment]
The duplicate IDs in the sidebar were fixed by https://github.com/flutter/website/commit/4140d207a6e6fc9283d9defd58dc40285412db94. Let me know if you come across any other cases. Thanks!
